assets/app/view/welcome.rb

```diff
@@ -18,6 +18,7 @@ def render_notification
       message = <<~MESSAGE
         <p><a href='https://www.kickstarter.com/projects/1840vienna/18esp/'>18ESP is now on Kickstarter</a>.</p>
         <p><a href="https://github.com/tobymao/18xx/wiki/18Neb">18Neb</a> is in alpha.</p>
+        <p><a href="https://github.com/tobymao/18xx/wiki/18NL">1847 AE</a> is in production.</p>
```

Comment:
Wrong URL for wiki page.
```suggestion
        <p><a href="https://github.com/tobymao/18xx/wiki/1847-AE">1847 AE</a> is in production.</p>
```

Reply:
Thanks, fixed.